pkg/cli/cli.go

```diff
@@ -235,6 +263,9 @@ func doRequest(client *http.Client, method, url string, reqPayload interface{},
 	defer res.Body.Close()
 	dec := json.NewDecoder(res.Body)
 	if res.StatusCode < 200 || res.StatusCode > 299 {
+		if method == "DELETE" {
+			return &apiCallError{&apiv1.Error{Code: strconv.Itoa(res.StatusCode)}}
```

Comment:
Would it make sense to use `res.Status` instead of `strconv.Itoa(res.StatusCode)`. The status will have more information than the code, the downside is that it won't be easy to parse back into an int later.

Reply:
done. Good point. Uses the message property instead `res.Status`